nires/removenoise.py

```diff
@@ -9,12 +9,10 @@
 from astropy.io import fits
 #title = ''
 
-#title,prefix=imD.returnInst(sys.argv[1])
-name=imD.nameResolve(sys.argv[2],'s*')
+#print sys.argv[2]
+#name=imD.nameResolve(sys.argv[2],'s*')
 
-
-
-noise_image = fits.open(name)
+noise_image = fits.open('s151112_a000017.fits')
 
 noise_data = noise_image[0].data
 total = np.zeros(2048)
```

Comment:
There are lots of hard coded variables here, lets parameterize them, and put all the parameters at the top of the code. Also instead of hard coding 2048 get the size of the image and use that here.